lib/hllcounter.cc

```diff
@@ -348,7 +348,7 @@ uint64_t HLLCounter::estimate_cardinality()
 void HLLCounter::add(const std::string &value)
 {
     HashIntoType x = khmer::_hash_murmur(value);
-    uint64_t j = x & (this->m - 1);
+    uint64_t j = x.as_ull() & (this->m - 1);
```

Comment:
@camillescott what is the right thing to do here? `x` can now be >64bit but we want to store it in a `uint64` 😔